windows/ReactTestApp/ReactInstance.cpp

```diff
@@ -57,12 +65,36 @@ namespace
     constexpr bool kUseCustomDeveloperMenu = true;
 
     REACT_MODULE(DevSettings)
-#endif
+#endif  // REACT_NATIVE_VERSION > 0 && REACT_NATIVE_VERSION < 6305
     struct DevSettings {
         REACT_INIT(Initialize)
-        void Initialize(winrt::Microsoft::ReactNative::ReactContext const &reactContext) noexcept
+        void Initialize(ReactContext const &reactContext) noexcept
         {
             context_ = reactContext;
+#if __has_include(<JSI/JsiApiContext.h>)
+            // There is currently no way to hook into when a JS bundle is done
+            // loading. However, it seems that by posting a callback, we can
+            // defer fetching app keys until everything is done executing.
+            reactContext.JSDispatcher().Post([reactContext]() noexcept {
+                auto &runtime =
```

Comment:
What happens if you get here when webdebugging?

Reply:
This is a good call. The test app is broken due to an unrelated issue fixed in #493.

With the fix applied, what happens is that we return early with an empty array here:
https://github.com/microsoft/react-native-test-app/blob/09fca861f0943e82b7890aad88ad4204150364b0/common/AppRegistry.cpp#L22-L25

I'm hesitant to implement a fallback for this, especially when web debugging might go away soon. Unless you have any suggestions, I'm going to leave this as is.